cmd/argocd/commands/project.go

```diff
@@ -839,7 +844,11 @@ func formatOrphanedResources(p *v1alpha1.AppProject) string {
 	if p.Spec.OrphanedResources == nil {
 		return "disabled"
 	}
-	return fmt.Sprintf("enabled (warn=%v)", p.Spec.OrphanedResources.IsWarn())
+	details := fmt.Sprintf("warn=%v", p.Spec.OrphanedResources.IsWarn())
+	if len(p.Spec.OrphanedResources.Ignore) > 0 {
+		details = fmt.Sprintf("%s, ignored %d", details, len(p.Spec.OrphanedResources.Ignore))
```

Comment:
It took me a while to realize that the attribute spec.OrphanedResources.Ignore had got merged 2 days ago :)

Reply:
Argo CD documentation website does not support versioning, unfortunately. Is causes a lot of confusion. @jannfis has created PoC some time ago which migrated docs to another framework that supports versions. Maybe it is time to work on migration.